bokeh/io.py

```diff
@@ -37,6 +37,8 @@
 from .util.notebook import load_notebook, publish_display_data, get_comms
 from .util.string import decode_utf8
 from .util.serialization import make_id
+from .resources import INLINE
+from .core.templates import ZEPPELIN_FILE
```

Comment:
I plan to create `bokeh.util.jupyter` to replace `bokeh.util.notebook` as all the methods in `bokeh.util.notebook` are for jupyter for now. I will still keep `bokeh.util.notebook` as deprecated.  What do you think ?

Regarding `inline`, I think we can do it in another PR. Because I think one PR should just do one thing.